src/lib/storage/value_vector.hpp

```diff
@@ -0,0 +1,125 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "fixed_string.hpp"
+#include "types.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+// FixedStringVector is a data type, which stores all its values in a vector and
+// is capable of storing FixedStrings.
+
+class FixedStringVector {
```

Comment:
Please move the code to a cpp file and extract the class (`fixed_string_vector.hpp`)